dagatsoin/git-workshop
# [FEATURE] Add a progress arrow

**This PR does the following**
- adds an arrow above the todo input
- makes the arrow grow as you add some todos
- each section of the arrow has the color as the corresponding todo is complete or not

**QA**
- open `index.html`
- add two todos
- [ ] check the arrow has two grey `=`
- validate the first todo
- [ ] check the first `=` is green
